Refactor : eslint 검사 및 let 안쓰고 구현

alstjr0183 opened the pull request:
eslint 검사와 let을 사용하지 않고 과제를 구현하였습니다.
리뷰 요청 드리겠습니다!

alstjr0183 committed:
Refactor : eslint 검사 및 let 안쓰고 구현

eslint 검사와 let을 사용하지 않고 과제를 구현하였습니다.

passwd10 commented on src/index.js:
```diff
@@ -1,2 +1,63 @@
+  return element;
+}
+
+const obj = { count: 0 };
```
전역변수를 사용하지 않고 문제를 해결할 수 있을까요?

passwd10 replied:
전역변수를 사용하지 않고 문제를 해결할 수 있을까요?

passwd10 commented on src/index.js:
```diff
@@ -1,2 +1,63 @@
+  function handleClick() {
+    obj.count += 1;
+    render();
+  }
+  function handleClickNumber(value) {
```
공백을 추가해서 각 함수를 구분해주면 좋을 것 같아요!

passwd10 replied:
공백을 추가해서 각 함수를 구분해주면 좋을 것 같아요!

passwd10 reviewed (commented):
let을 사용하지 않고 과제를 해결하셨네요! 👍 
현재 const 객체 내부의 값을 재할당하고 있는데, 재할당하지 않고 문제를 해결하는 방법에 대해서 고민해보시면 좋을 것 같아요~

alstjr0183 committed:
fix 전역변수 사용하지않고 구현

render 함수에 매개변수로 count를준뒤 기본값 0을 설정한뒤 클릭이벤트들은 render함수 사용시 인자로 넘겨서 컨트롤하게끔 수정해보았습니다
마지막에 document.getElementById('app') 반복되는 부분이있길래 변수로 빼내어서 수정해보았습니다!
